src/styles/base/_base.scss

```diff
@@ -111,4 +111,18 @@ h2 {
     border-style: solid;
     border-color: var(--mat-table-row-item-outline-color, rgba(0, 0, 0, 0.12));
   }
-}
\ No newline at end of file
+}
+
+// Angular Material
+
+.btn-toogle-select {
```

Comment:
Maybe this should be in form css?

Reply:
Moved to form and added their sass variables